### PR TITLE
add kubectl diff to cheatsheet

### PR DESCRIPTION
**Pull request by zhouya0**
Add `kubectl diff` to cheatsheet as `kubectl diff` is already GA.
Part of https://github.com/kubernetes/kubernetes/issues/86525.

### CODE REVIEW

**Comment by apelisse** [comment]
Thanks a lot @zhouya0 !
/approve
/lgtm

File: content/en/docs/reference/kubectl/cheatsheet.md [hunk 1]
```diff
@@ -195,6 +195,9 @@ kubectl get pods -o json | jq '.items[].spec.containers[].env[]?.valueFrom.secre
 
 # List Events sorted by timestamp
 kubectl get events --sort-by=.metadata.creationTimestamp
+
+# Diff configurations with the current online configuration
```
Comment (jimangel) [inline content/en/docs/reference/kubectl/cheatsheet.md]: I think there can be a clearer description here but I'm not sure how to word it. My concern is mainly around the word "online" which can be confusing, especially if your cluster is on-prem.

Maybe replace w/ cluster-state ?

Comment (apelisse) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Let's be clear about what this does: it compares the current state of the cluster against the state that the cluster would be in if you had applied that configuration. I suspect with could almost use that sentence as-is.

```
# Compares the current state of the cluster against the state that the cluster would be in if the manifest was applied.
```

Comment (zhouya0) [reply content/en/docs/reference/kubectl/cheatsheet.md]: Thanks a lot @apelisse.  My native language is not english, to be honest i have to read multiple times with this sentence. But I aggree with the better explaination about  the `online cluster` :)

**Comment by zhouya0** [comment]
@jimangel please review this, thanks!

**Pull request merged**